packages/components/src/public/QueryModel/GridPanel.tsx

```diff
@@ -20,16 +20,14 @@ import {
 import { GRID_SELECTION_INDEX } from '../../internal/constants';
 import { DataViewInfo } from '../../internal/models';
 import { headerCell, headerSelectionCell, isFilterColumnNameMatch } from '../../internal/renderers';
-import { ActionValue } from '../../internal/components/omnibox/actions/Action';
-import { FilterAction } from '../../internal/components/omnibox/actions/Filter';
-import { SearchAction } from '../../internal/components/omnibox/actions/Search';
-import { SortAction } from '../../internal/components/omnibox/actions/Sort';
-import { ViewAction } from '../../internal/components/omnibox/actions/View';
-import { Change, ChangeType, OmniBox } from '../../internal/components/omnibox/OmniBox';
 
-import { isGridColSortFilterEnabled } from '../../internal/app/utils';
+import { ActionValue } from './grid/actions/Action';
+import { FilterAction } from './grid/actions/Filter';
+import { SearchAction } from './grid/actions/Search';
+import { SortAction } from './grid/actions/Sort';
+import { ViewAction } from './grid/actions/View';
```

Comment:
If these things move into the public directory, are there any public docs that need to be updated?

Reply:
No doc updates needed for these files/actions in particular. We have lots of internal QueryModel/GridPanel related components in /public which are not documented for external use. However, there was a usages of the "showOmnibox" prop for GridPanel which was referenced in the QueryModel.md doc page. I've updated that.